spec/definitions/Tags/Tag.yaml

```
@@ -0,0 +1,20 @@
+type: object
+description: Tag
+required:
+- name
+properties:
+  id:
+    description: The tag's identifier string
+    readOnly: true
+    allOf:
+    - $ref: "#/definitions/ResourceId"
+  name:
+    description: The tag's name
```

```suggestion
    description: The tag's name, which must be unique.
```

Shouldn't there be a max length here?

---

What about allowed pattern?